.github/workflows/frontend-ci.yml

```
@@ -0,0 +1,264 @@
+          END=$(date +%s)
+          DURATION=$((END - START))
+          SIZE=$(du -sh .nuxt/dist | cut -f1)
+          echo "duration=${DURATION}s" >> $GITHUB_OUTPUT
+          echo "size=${SIZE}" >> $GITHUB_OUTPUT
+        working-directory: ${{ env.WORKING_DIRECTORY }}
+        env:
+          NODE_ENV: production
+          BACKEND_URL: https://api.ai-manus.com
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.8)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)